resources/jira/nodered_flows.json

```diff
@@ -0,0 +1,194 @@
+        "output": "str",
+        "x": 410,
+        "y": 320,
+        "wires": [
+            [
+                "493ad57a3a60ef52"
+            ]
+        ]
+    },
+    {
+        "id": "1d7cd8300af4ce47",
+        "type": "inject",
+        "z": "6d8a75918fd777a0",
+        "name": "",
+        "props": [
+            {
+                "p": "payload"
+            },
+            {
+                "p": "topic",
+                "vt": "str"
+            }
+        ],
+        "repeat": "",
+        "crontab": "",
+        "once": true,
+        "onceDelay": 0.1,
+        "topic": "",
+        "payloadType": "date",
+        "x": 120,
+        "y": 320,
+        "wires": [
+            [
+                "1e5a404c574f3ad7"
+            ]
+        ]
+    },
+    {
+        "id": "b1c40c906ba60e24",
+        "type": "exec",
+        "z": "6d8a75918fd777a0",
+        "command": "API_KEY=${API_KEY} API_USR=${API_USR} PRJ=31 python3 ./monitor_ticket_progress.py",
+        "addpay": "",
+        "append": "",
+        "useSpawn": "false",
+        "timer": "",
+        "winHide": false,
+        "oldrc": false,
+        "name": "Run monitor script",
+        "x": 450,
+        "y": 180,
+        "wires": [
+            [
+                "5d618cc997f9f79b"
+            ],
+            [
+                "3eead32efe2982f5"
+            ],
+            []
+        ]
+    },
+    {
+        "id": "214dddfc36532b95",
+        "type": "e-mail",
+        "z": "6d8a75918fd777a0",
+        "server": "server2.hosting2go.nl",
```

Comment:
Is this referring to a personal server of some kind?

Reply:
Yes, it is, it's my email provider's SMTP server.

Reply:
So this will cause breakage if you were to leave the team? I'm not really sure what it says here.

Reply:
Yes, but it's provided as an example.
UM currently has no node-red setup available. If this kind of reporting yields nice results. I'll put in a request and de-couple it from my specifics.

Reply:
I'll make the nature of this script clear in the README.md

Reply:
As to what it says: references to local pwd storage or s'thing. You can still import the flow, but for it to work, you'll of course need to change some settings.